packages/common.vm/tools/vm.common/vm.common.psm1

```diff
@@ -1358,8 +1358,18 @@ function VM-Sort-Desktop-Icons {
     VM-Write-Log "INFO" "Sorting Desktop icons"
     (New-Object -ComObject Shell.Application).toggleDesktop();
     Start-Sleep -Milliseconds 100;
+    # Start Windows Run
```

Comment:
why do we need to start Windows Run?

Reply:
With that pop-up, I'm about to use `Alt-F4` to simply close it, which will put the focus onto the desktop.  There may be other ways, but I couldn't find a better solution. 😅 

It may also work with other window pop-ups, but this was a simple one to start. Other ones I tried needed the Windows key + <something> to open and that take as easy to emulate with the SendKeys functionality.